makefile

```diff
@@ -162,19 +160,19 @@ set-smoke-git-config:
 	git config user.email email
 load-smoke-data:$(buildDir)/.load-smoke-data
 load-local-data:$(buildDir)/.load-local-data
-$(buildDir)/.load-smoke-data:$(buildDir)/load-smoke-data
-	./$<
+$(buildDir)/.load-smoke-data:$(buildDir)/.get-mongotools smoke/internal/testdata/mongodump
+	$(call run-mongotools,mongorestore) --drop smoke/internal/testdata/mongodump
```

Comment:
nit: I know how this works, but it would be helpful for other maintainers who aren't as familiar with Make to explain that this works by invoking the `run-mongotools` target with the `mongorestore` argument.

Reply:
Done.